loopflow-computation/pom.xml

```diff
@@ -41,7 +41,11 @@
         </dependency>
         <dependency>
             <groupId>com.powsybl</groupId>
-            <artifactId>powsybl-sensitivity-api</artifactId>
+            <artifactId>powsybl-sensitivity-analysis-api</artifactId>
```

Comment:
I think there is no need of this module here

Reply:
It doesn't appear here but in this pom I think you can also remove iidm-impl and loadflow-api

Reply:
Needed for parameters ...